scripts/start-tagger.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
If the file is a docker resource, then it should be placed in docker-resources

Reply:
My bad, it's already there. I'll remove it.